utils/build_system/create_test_release_jobs.py

```diff
@@ -18,12 +24,12 @@ def create_directory(self, name, display_name):
             dir_xml_data = DIR_TEMPLATE % dict(sct_display_name=display_name)
             self.jenkins.create_job(f'{self.base_job_dir}/{name}', dir_xml_data)
         except jenkins.JenkinsException as ex:
-            print(ex)
+            LOGGER.info(ex)
 
     def create_pipeline_job(self, jenkins_file, group_name, job_name=None, job_name_suffix="-test"):
         base_name = job_name or Path(jenkins_file).stem
         sct_jenkinsfile = jenkins_file.split("scylla-cluster-tests/")[-1]
-        print(sct_jenkinsfile)
+        LOGGER.info(sct_jenkinsfile)
```

Comment:
Make it more informational... or change to DEBUG

Reply:
fixed